AdobeStockImageAdminUi/composer.json

```diff
@@ -1,16 +1,20 @@
 {
     "name": "magento/module-adobe-stock-image-admin-ui",
-    "description": "N/A",
+    "description": "Magento module responsible for the admin panel images UI implementation",
     "require": {
         "php": "~7.1.3||~7.2.0||~7.3.0",
         "magento/framework": "*",
-        "magento/module-ui": "*",
-        "magento/module-backend": "*",
-        "magento/module-adobe-stock-image-api": "*",
-        "magento/module-adobe-stock-asset-api": "*",
         "magento/module-adobe-ims": "*",
         "magento/module-adobe-ims-api": "*",
-        "magento/module-adobe-ui": "*"
+        "magento/module-adobe-stock-asset": "*",
```

Comment:
`magento/module-adobe-stock-image`, `magento/module-adobe-stock-asset` and `magento/module-adobe-stock-ims` should be eliminated from `AdobeStockImageAdminUi` module, it should rely on corresponding `Api` modules instead